datacite-findable-doi-handler/src/main/java/no/unit/nva/datacite/handlers/LandingPageUtil.java

```diff
@@ -0,0 +1,42 @@
+package no.unit.nva.datacite.handlers;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+
+public class LandingPageUtil {
+    static final String PUBLICATION_LANDING_PAGE_COULD_NOT_BE_CONSTRUCTED = "Landing page could not be "
+        + "constructed";
+    public static final String LANDING_PAGE_PATH_FORMAT = "/registration/%s/public";
+
+    private LandingPageUtil() {
+    }
+    protected static URI getLandingPage(URI publicationId) {
+        return Optional.of(publicationId)
+            .map(LandingPageUtil::buildLandingPageUrl)
+            .orElseThrow(() -> new IllegalArgumentException(PUBLICATION_LANDING_PAGE_COULD_NOT_BE_CONSTRUCTED));
+    }
+
+    static URI buildLandingPageUrl(URI publicationId) {
+        String path = publicationId.getPath();
+        int beginIndex = path.lastIndexOf("/");
```

Comment:
make the '/'  a public static constant that you can use everywhere in the project?

Reply:
Probably should be moved into StringUtils in commons or something? But to avoid having another thing to fix for the user story, it is only introduced as a static char inside the class.